css/barbill.css

```diff
@@ -1,137 +1,137 @@
 /* Google font assignment */
 
 body {
-    font-family: "Oswald", sans-serif;
-    background-color: #2A53A280;
+	font-family: "Oswald", sans-serif;
+	background-color: #2A53A280;
```

Comment:
I'm _pretty sure_ this should be `#2A53A2` Hex only uses 6 characters...

Reply:
Clarification: our goal was to consolidate all of the CSS into a single file bb_styles. While doing this there was a conflict with the menu pages css so that was kept as a separate file.
When completed we want the CSS down to only 3 files:
bb_common - includes styling for common elements that appear on every page (header, nav bar, footer)
menu - styling for the menu and take out pages
bb_styles - currently has styling for home and about us pages. Margaret will be adding the rest of the pages once this has been merged.
This will eliminate all other CSS files